superset-frontend/src/explore/components/controls/HiddenControl.tsx

```diff
@@ -17,29 +17,14 @@
  * under the License.
  */
 import React from 'react';
-import PropTypes from 'prop-types';
 import { Input } from 'src/common/components';
 
-const propTypes = {
-  onChange: PropTypes.func,
-  value: PropTypes.oneOfType([
-    PropTypes.string,
-    PropTypes.number,
-    PropTypes.object,
-    PropTypes.bool,
-    PropTypes.array,
-    PropTypes.func,
-  ]),
-};
-
-const defaultProps = {
-  onChange: () => {},
-};
+interface HiddenControlsProps {
+  onChange: () => void;
+  value: 'string | number | readonly string[] | undefined';
```

Comment:
this probably shouldn't be a string

```suggestion
  value: string | number | readonly string[] | undefined;
```

Reply:
That's right. Thanks for the suggestion.